### PR TITLE
add meta column and update apsan

### PR DESCRIPTION
**Pull request by farshadth**

### CODE REVIEW

File: src/Apsan/Apsan.php [hunk 1]
```diff
@@ -108,9 +110,14 @@ protected function sendPayRequest()
                 throw new ApsanException($response->description, $statusCode);
             }
 
-            $this->refId = $response->result;
+            $this->token = $response->result;
+            $this->refId = $this->uniqueIdentifier;
```
Comment (m-jch) [inline src/Apsan/Apsan.php:114]: Please use `$this->transactionId()` instead of `uniqid()` in Apsan gateway

File: src/DataBaseManager.php [hunk 2]
```diff
@@ -115,6 +115,7 @@ protected function createTables()
 					`price` decimal(15,2) NOT NULL,
 					`ref_id` varchar(1024) COLLATE utf8_persian_ci DEFAULT NULL,
 					`tracking_code` varchar(50) COLLATE utf8_persian_ci DEFAULT NULL,
+                    `meta` JSON DEFAULT NULL,
```
Comment (m-jch) [inline src/DataBaseManager.php:118]: Please fix the spacing

**Review by m-jch — CHANGES_REQUESTED** [review]

File: src/Apsan/Apsan.php (hunk 1 above)
Comment (farshadth) [reply src/Apsan/Apsan.php]: done

File: src/DataBaseManager.php (hunk 2 above)
Comment (farshadth) [reply src/DataBaseManager.php]: done

**Comment by farshadth** [comment]
all done

**Review by m-jch — APPROVED** [review]

**Pull request merged**